release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3356,6 +3356,46 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-16-15_{context}"]
+=== RHSA-2024:7174 - {product-title} {product-version}.15 bug fix and security update
+
+Issued: 2 October 2024
+
+{product-title} release {product-version}.15 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7174[RHSA-2024:7174] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:7177[RHBA-2024:7177] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.15 --pullspecs
+----
+
+[id="ocp-4-16-15-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a local patch to kube-proxy caused OpenShift SDN to add another duplicate copy of a particular rule to the iptables ruleset every time it resynchronized. The synchronization would slow down and eventually trigger the `NodeProxyApplySlow` alert. With this release, the kube-proxy patch has been fixed and the alert no longer appears. (link:https://issues.redhat.com/browse/OCPBUGS-42159[*OCPBUGS-42159*])
+
+* Previously, when the Node Tuning Operator (NTO) was configured using PerformanceProfiles it would create an ocp-tuned-one-shot systemd service. The systemd service would run prior to kubelet and blocked execution. The systemd service invokes Podman which uses an NTO image, but when the NTO image was not present Podman still tried to fetch the image and it would fail. With this release, support is added for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env`. Now, Podman pulls NTO images in environments which need to use proxies for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-42061[*OCPBUGS-42061*])
```

Comment:
```suggestion
* Previously, when the Node Tuning Operator (NTO) was configured by using PerformanceProfiles, it would create an ocp-tuned-one-shot systemd service. The systemd service would run prior to kubelet and blocked execution. The systemd service invokes Podman, which uses an NTO image. But, when the NTO image was not present, Podman still tried to fetch the image and it would fail. With this release, support is added for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env`. Now, Podman pulls NTO images in environments which need to use proxies for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-42061[*OCPBUGS-42061*])
```